pom.xml

```diff
@@ -456,7 +456,7 @@
         </repository>
     </repositories>
     <properties>
-        <swagger-parser-version>1.0.9-SNAPSHOT</swagger-parser-version>
+        <swagger-parser-version>1.0.9</swagger-parser-version>
```

Comment:
My changes don't depend on using swagger parser 1.0.9 instead of 1.0.9-SNAPSHOT; I made that change because the Travis CI failed to build until I bumped the version of swagger-parser to 1.0.9. I could change it back if necessary.

![screen shot 2015-08-06 at 10 48 44 am](https://cloud.githubusercontent.com/assets/10627244/9118973/f654f96a-3c28-11e5-99b5-93331bf4c5cc.png)